lib/school.rb

```diff
@@ -0,0 +1,21 @@
+class School
+    def initialize(time, hours)
+      @start_time = time
+      @hours = hours
+      @student_names = []
+    end
```

Comment:
This is a good start, but you will want to create some attr_readers for each of these, so that we can call things like `school.start_time`.  Because we want to call these attributes as methods, they will need to be named precisely what we ask for:

```ruby
class School
  attr_reader :start_time, :hours_in_school_day, :student_names
  
  def initialize(start_time, hours_in_school_day)
    @start_time = start_time
    @hours_in_school_day = hours_in_school_day
    @student_names = []
  end
end
```